pkg/controller/storagecluster/quickstarts.go

```diff
@@ -0,0 +1,79 @@
+package storagecluster
+
+import (
+	"bytes"
+	"context"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	k8sYAML "k8s.io/apimachinery/pkg/util/yaml"
+	"os"
+	"path"
+)
+
+var (
+	quickstartDir = "/quickstarts/"
+)
+
+func (r *ReconcileStorageCluster) ensureQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
```

Comment:
This also needs to ensure that the OCP cluster has the required CRDs and skip if it doesn't.
In OCP 4.6 where we don't have quickstart CRD, this would cause unwanted noise.

Reply:
as discussed, this requires a bunch of permissions and apis added to the ocs scheme. Will track this in a follow up